packages/dependency-extraction-webpack-plugin/lib/index.js

```diff
@@ -278,12 +278,32 @@ class DependencyExtractionWebpackPlugin {
 			const { hashFunction, hashDigest, hashDigestLength } =
 				compilation.outputOptions;
 
-			const contentHash = chunkFiles
-				.sort()
-				.reduce( ( hash, filename ) => {
-					const asset = compilation.getAsset( filename );
-					return hash.update( asset.source.buffer() );
-				}, createHash( hashFunction ) )
+			const hashBuilder = createHash( hashFunction );
+
+			const processContentsForHash = ( content ) => {
+				hashBuilder.update( content );
+			};
+
+			// Prepare to look for magic comments, in order to decide whether
+			// `wp-polyfill` is needed.
+			const processContentsForMagicComments = ( content ) => {
+				if ( content.toString().includes( '/* wp:polyfill */' ) ) {
```

Comment:
It looks like content is a `Buffer` and [`Buffer` has an `includes` method](https://nodejs.org/api/buffer.html#bufincludesvalue-byteoffset-encoding). Do we need to call `toString`?

```suggestion
				if ( content.includes( '/* wp:polyfill */' ) ) {
```

Reply:
That does seem to work! Thanks 🙂